app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -0,0 +1,547 @@
+import { stratify, tree as d3Tree } from "d3-hierarchy";
+import { select, event as currentEvent } from "d3-selection";
+import { scaleLinear } from "d3-scale";
+
+export default class TidyTree {
+  constructor(container, nodes, options) {
+    this.container = select(container);
+    this.svg = null;
+    this.pathContainer = null;
+    this.nodeContainer = null;
+
+    this.options = Object.assign(
+      {
+        addOverlays: true,
+        attribute: "aggregatescore",
+        leafNodeHeight: 35,
+        minWidth: 1000,
+        minHeight: 300,
+        transitionDuration: 500,
+        onNodeHover: null,
+        onCreatedTree: null,
+        tooltipContainer: null,
+        collapseThreshold: 0.4,
```

Comment:
It would be amazing if you could take this as a prop coming straight from the database, because we might want to tweak it for demos on short notice without deploying code changes. There's a table called `ui_configs`

Reply:
Maybe, I would prefer to make a UI choice with a slider or something.

Reply:
that would also be cool